src/offset/mod.rs

```diff
@@ -46,6 +46,21 @@ pub use self::utc::Utc;
 ///   DST.
 /// * No result when the clock is turned forwards during a transition due to for example DST.
 ///
+/// <div class="warning">
+///
+/// In wasm, when using [`Local`], only the [`LocalResult::Single`] variant is returned.
+/// Specifically:
+///
+/// * When the clock is turned backwards, where `Ambiguous(earliest, latest)` would be expected,
+///   `Single(earliest)` is returned instead.
+/// * When the clock is turned forwards, where `None` would be expected, `Single(t)` is returned,
+///   with `t` being the requested local time represented as though there is no transition on that
+///   day (i.e. still "summer time")
+///
+/// See further discussion and workarounds in <https://github.com/chronotope/chrono/issues/1701>.
```

Comment:
Can you add a sentence or two on why/how this happens? I think that would be helpful.

Reply:
I'd love to, but I couldn't really figure it out from a glance at the code. Could you point me in the right direction for "where do we actually ask the browser to do time conversion"?

Reply:
Presumably here?

https://github.com/chronotope/chrono/blob/main/src/offset/local/mod.rs#L61

Reply:
Ah, well, yep, it's in there all right. But it's kinda tautological:

https://github.com/chronotope/chrono/blob/361b9fc61eb9d495b41fa8de6296866323cb53e7/src/offset/local/mod.rs#L90-L91

Reply:
I guess it's fine to say that browsers always provide a single offset? Maybe even link to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getTimezoneOffset for details.

Reply:
Maybe once we get [Temporal](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Temporal#browser_compatibility) we can do better in WASM?